net/redisclient.go

```diff
@@ -197,6 +199,19 @@ func NewRendezvousVnodes(shards []string) redis.ConsistentHash {
 	return rendezvousVnodes{rendezvous.New(vshards, xxhash.Sum64String), table}
 }
 
+func (rrc RedisRingClient) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
TH GH action should be solved by using a pointer receiver instead of the value receiver.